server/proxy/tcpproxy/userspace.go

```diff
@@ -69,13 +70,29 @@ type TCPProxy struct {
 	pickCount int // for round robin
 }
 
+// The parameter host is returned by net.SplitHostPort previously,
+// so it must be a valid host. This function is only to check whether
+// it's an IPv6 IP address.
+func isIPv6(host string) bool {
+	return strings.IndexByte(host, byte(':')) >= 0
```

Comment:
```suggestion
	return strings.IndexRune(host, ':') != -1
```

Reply:
Thanks. I agree that IndexRune is better. Based on the golang spec: `A rune literal represents a rune constant, an integer value identifying a Unicode code point`

Fixed.